internal/v1/server_test.go

```diff
@@ -424,6 +424,50 @@ func TestGetComposeMetadata404(t *testing.T) {
 	require.Contains(t, body, "Compose not found")
 }
 
+func TestGetComposes(t *testing.T) {
+	composes := ComposesResponse{
+		Data: []ComposesResponseItem{
+			{
+				CreatedAt: "",
+				Id:        UUIDTest,
+			},
+		},
+	}
+
+	api_srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.Header().Set("Content-Type", "application/json")
+
+		err := json.NewEncoder(w).Encode(composes)
+		require.NoError(t, err)
+
+	}))
+	defer api_srv.Close()
```

Comment:
for `/composes` you don't need the api_srv, just pass an empty string to startServer.